ban/core/resource.py

```diff
@@ -126,25 +128,54 @@ def as_version(self):
         """Resources plus relations references and metadata."""
         return self.serialize({f: {} for f in self.versioned_fields})
 
+    @property
+    def status(self):
+        return 'deleted' if self.deleted_at else 'active'
+
+    @classmethod
+    def active(cls):
```

Comment:
Indeed, but safer. Not that this is critical but still you (or someone else) don't want to forget to use the `active()` filter, do you?